projects/core/src/lib/layers/legend/legend-raster/raster-legend.component.ts

```diff
@@ -49,8 +42,8 @@ export function calculateNumberPipeParameters(breakpoints: Array<ColorBreakpoint
     pure: true,
 })
 export class CastMeasurementToClassificationPipe implements PipeTransform {
-    transform(value: unknown, _args?: unknown): ClassificationMeasurement | null {
-        if (value instanceof ClassificationMeasurement) {
+    transform(value: Measurement, _args?: unknown): ClassificationMeasurement | null {
+        if (value.type == 'classification') {
```

Comment:
Vielleicht kann man für die Pipe hier einen Unit-Test in ein Spec-File schreiben. Das wäre doch einer der raren Momente, bei denen wir hier prüfen können, ob das funktioniert.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/749/commits/05e473a1a037a44a13408f00b1c599ca532bdcf7